library/src/main/java/com/datatorrent/lib/io/fs/FileSplitterInput.java

```diff
@@ -452,7 +447,7 @@ private void scan(Path filePath, Path rootPath, Map<String, Long> lastModifiedTi
           }
         }
       } catch (FileNotFoundException fnf) {
-        LOG.warn("Failed to list directory {}", filePath, fnf);
+        LOG.error("Failed to list directory {}", filePath, fnf);
```

Comment:
I agree. And if the idea was to support cases where a directory gets created after the app is already running, there should maybe be a property to control it (ignoreMissingDir).